frontend/src/pages/CourseSelector/courseDescription/courseDescription.less

```diff
@@ -90,6 +90,20 @@
   gap: 20px;
 }
 
+.progressBarWrapper {
+  width: '100%';
+  background-color: 'whitesmoke';
```

Comment:
Can you specify the hex code instead? Also, for `border-radius` could you specify the unit of measurement aswell? i.e. `px`